README.md

```diff
@@ -13,7 +13,38 @@ To run review of specific delivery execution use the next endpoint:
 https://YOUR_DOMAIN/ltiTestReview/ReviewTool/launch?execution=YOUR_DELIVERY_EXECUTION_URI
 ```
 
-Endpoint without `execution` parameter (`https://YOUR_DOMAIN/ltiTestReview/ReviewTool/launch`) will use `lis_result_sourcedid` field from lauch data to determine delivery execution.
+Endpoint without `execution` parameter (`https://YOUR_DOMAIN/ltiTestReview/ReviewTool/launch`) will use `lis_result_sourcedid` field from launch data to determine delivery execution.
+
+To run review of the latest delivery execution for user use next endpoint:
+
+```
+https://YOUR_DOMAIN/ltiTestReview/ReviewTool/launch1p3?delivery=YOUR_DELIVERY_URI
+```
+
+User id should be provided by usage `for_user` claim
+
+```json
+{
```

Comment:
Message type must also be specified here.
Could you please add it?

`"https://purl.imsglobal.org/spec/lti/claim/message_type": "LtiSubmissionReviewRequest"`

Reply:
sure

Reply:
done